VoodooI2CELAN/VoodooI2CELANTouchpadDriver.hpp

```diff
@@ -176,6 +180,8 @@ class VoodooI2CELANTouchpadDriver : public IOService {
      * @return kIOSuccess if the message is processed
      */
     IOReturn message(UInt32 type, IOService* provider, void* argument) override;
+    
+    void simulateInterrupt(OSObject* owner, IOTimerEventSource* timer); // Sasha - impl polling support
```

Comment:
Please remove the  @EmotionalLove's comment here.

Reply:
fixed